src/utils/pause/mapScopes/index.js

```diff
@@ -298,12 +314,23 @@ function generateClientScope(
   return result;
 }
 
+function hasValidIdent(range: MappedOriginalRange, pos: BindingLocation) {
+  return (
+    range.type === "match" ||
+    // For declarations, we allow the range on the identifier to be a
+    // more general "contains" to increase the chances of a match.
+    (pos.type !== "ref" && range.type === "contains")
+  );
+}
```

Comment:
what is this?

Reply:
I can try to give it a better name if that would help. This makes sure that the original range only contains a single binding, and if it's a reference and not a declaration, it also requires that the binding be at the start of the range.